src/components/Auth/AuthForm.tsx

```diff
@@ -1,4 +1,4 @@
-import classNames from 'classnames';
+import cn from 'classnames';
```

Comment:
```suggestion
import classNames from 'classnames';
```
Better to use full name. `cn` may confuse other developer